modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/configuration.jsp

```diff
@@ -92,6 +92,12 @@ DLPortletInstanceSettingsHelper dlPortletInstanceSettingsHelper = new DLPortletI
 				<div class="form-group">
 					<aui:input label="root-folder" name="rootFolderName" type="resource" value="<%= dlAdminDisplayContext.getRootFolderName() %>" />
```

Comment:
We also need to update the folder name. If the folder is in trash, it will show a temporary name (e.g. `/4024`); we should retrieve the trashed entry and display its original name.